### ISSUE TITLE
Crossref deposit message inaccurate when DOIs registered manually

### ISSUE THREAD

**Issue by NateWr** [pkp/pkp-lib#7521]
**Describe the problem you would like to solve**
When a DOI has a registered status in the new DOI management UI (#6062), the following message is shown from the registration agency (eg - Crossref):

> The metadata for this item has been submitted to Crossref.

This might be misleading if the item was manually marked registered. The application only know that the DOI is registered somewhere, it doesn't know that it's registered at Crossref or that anyone ever actually completed the manual registration.

**Describe the solution you'd like**
If possible, the status message shown for the registration agency should distinguish between manual registration and registration/deposit through OJS/OMP/OPS. When manual registration has occurred, show an appropriate message like:

> This item has been manually registered with a registration agency.

Perhaps this can be done by having the registration agency identify whether or not a status message from the registration agency has been recorded for the DOI settings. That would indicate whether or not the application had made the deposit.

**Who is asking for this feature?**
This is part of #6062.

**Additional information**
cc @ewhanson

**Comment by ewhanson** [comment]
PRs:

- `pkp-lib`: #8494
- `ui-library`: pkp/ui-library#240
- `crossref-ojs`: pkp/crossref-ojs#20
- `ojs`: pkp/ojs#3663

**Comment by ewhanson** [comment]
Hi @bozana, could you have a look at this? Thanks!

PRs:

- `pkp-lib`: #8494
- `ui-library`: pkp/ui-library#240
- `crossref-ojs`: pkp/crossref-ojs#20
- `crossref-ops`: pkp/crossref-ops#19
- `ojs`: pkp/ojs#3663
- `omp`: pkp/omp#1284
- `ops`: pkp/ops#420

**Comment by bozana** [comment]
@ewhanson, just two questions in ui-library.
What happens when a user changes the registration agency? Would the information displayed in the DOI management page and DOI settings work correctly in that case? -- I haven't test it... For example, if the user re-register the DOIs with the new agency...
And one more thing: we would need to provide upgrade for MEDRA as well -- no matter if the plugin is adapted or not... So maybe to create a new issue just for that?
Thanks a lot!

**Comment by ewhanson** [comment]
Thanks @bozana.

To your questions:

- I'm not completely sure what would happen, but that sounds correct. The deposit status is on a DOI-basis, not a registration agency basis, so if you were to change the registration agency then attempt to "re-deposit" it, it should then appear in the systems as an error (because the prefix is wrong or whatever else the case may be).
- I'm not sure what the best practice around different prefixes would be. I know with hosting, we come across journals that have been around in different iterations and may have different prefixes across all of their publications, so I don't know that we could reliably determine if it should work with a given registration agency based on the prefix.
- That sounds good on MEDRA. I've created an issue to track that work here: #8597.

**Comment by bozana** [comment]
Thanks @ewhanson. Once this is ready it can be merged...

**Comment by ewhanson** [comment]
Merged!
